torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -98,6 +98,9 @@ bool validateKernelArgTensor(
     case at::ScalarType::Long:
```

Comment:
Should we do a follow up to make our DataType consistent with theirs? Long/Int vs Int/Int32

Reply:
Sure. It'd be nice. I will do that in a follow up PR.